data/chameleoncloud/sites/uc/clusters/chameleon/nodes/16035a48-fc53-4b5f-87fc-f911569545da.json

```diff
@@ -5,91 +5,93 @@
     "smt_size": 96
   },
   "bios": {
-    "release_date": "02/24/2021",
-    "vendor": "Dell Inc.",
+    "release_date": "2021-02-24",
+    "vendor": "Dell",
     "version": "2.10.2"
   },
   "chassis": {
-    "manufacturer": "Dell Inc.",
+    "manufacturer": "Dell",
     "name": "PowerEdge R740",
     "serial": "76G4KD3"
   },
+  "gpu": {
+    "gpu": false
+  },
   "main_memory": {
     "humanized_ram_size": "192 GiB",
-    "ram_size": 192000000000
+    "ram_size": 206158430208
   },
   "monitoring": {
     "wattmeter": false
   },
   "network_adapters": [
     {
-      "device": "NIC.Integrated.1-1",
+      "device": "eno1np0",
+      "driver": "mlx5_core",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "b8:ce:f6:43:21:6e",
-      "management": false,
-      "model": "MLNX 25GbE 2P ConnectX4LX RNDC",
+      "model": "MT27710 Family [ConnectX-4 Lx]",
       "rate": 25000000000,
-      "vendor": "Mellanox Technologies"
+      "vendor": "Mellanox"
     },
     {
-      "device": "NIC.Integrated.1-2",
+      "device": "eno2np1",
+      "driver": "mlx5_core",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "b8:ce:f6:43:21:6f",
-      "management": false,
-      "model": "MLNX 25GbE 2P ConnectX4LX RNDC",
+      "model": "MT27710 Family [ConnectX-4 Lx]",
       "rate": 25000000000,
-      "vendor": "Mellanox Technologies"
+      "vendor": "Mellanox"
     },
     {
-      "device": "NIC.Slot.7-1",
+      "device": "ens7f0",
+      "driver": "tg3",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "e4:3d:1a:2d:db:62",
-      "management": false,
-      "model": "BRCM GbE 2P 5720-t Adapter",
-      "vendor": "Broadcom Inc. and subsidiaries"
+      "model": "NetXtreme BCM5720 Gigabit Ethernet PCIe",
+      "rate": 1000000000,
```

Comment:
"rate" is something that was difficult to get out-of-band, I believe ironic inspection is fetching this from ethtool.

This rate is in bits-per-second, but we probably want to indicate something more user friendly on the frontend, say 1gbps, 10g, 25g , 100g, ...

There are also two different values to be concerned with, and I'm unsure which is reported here:

- the maximum rate the card can negotiate, say 25gb/s
- the rate the card can actually do with its current connection, which could be 10gb/s depending on the switch and cable